src/server/pachyderm_test.go

```diff
@@ -4016,7 +4016,6 @@ func TestManyLogs(t *testing.T) {
 
 func TestLokiLogs(t *testing.T) {
 	// TODO(2.0 required): This test is taking too long to complete due to slow pipelines
```

Comment:
We can probably remove this if it is no longer flaky and we suspect the issue was a delay in Loki.

Reply:
Removed, the test took an average of ~14.5s to run on my machine which seems pretty reasonable. I suspect a big part of the slowness was the fact that the test had a 30 second sleep at the end for no particular reason.